scripts/ci_header_doc_coverage.rb

```
@@ -0,0 +1,430 @@
+#!/usr/bin/env ruby
```

RuboCop (1.84.1)

[warning] Script file ci_header_doc_coverage.rb doesn't have execute permission.

(Lint/ScriptPermission)